packages/flutter/lib/src/rendering/object.dart

```diff
@@ -624,7 +624,7 @@ class _RootSemanticsFragment extends _InterestingSemanticsFragment {
     assert(parentSemantics == null);
     owner._semantics ??= new SemanticsNode.root(
       handler: owner is SemanticActionHandler ? owner as dynamic : null,
-      owner: owner.owner
+      owner: owner.owner.semanticsOwner
```

Comment:
not sure whether to suggest that we rename the two `owner`s, or whether to suggest that `semanticsOwner` should be called `owner`.